SunEngine/Program.cs

```diff
@@ -40,66 +34,17 @@ public static void Main(string[] args)
             }
             else
             {
-                if (SunEngineDllRunServer(startupConfiguration))
-                    RunServer(startupConfiguration);
-                else
-                    InfoPrinter.PrintVoidStartInfo();
+                ServerStartup.RunServer(startupConfiguration);    
             }
+            
         }
 
         private static bool ShouldUpdateData(StartupConfiguration startupConfiguration)
         {
-            return startupConfiguration.Migrate || 
+            return startupConfiguration.Migrate ||
                    startupConfiguration.InitializeCoreData ||
                    startupConfiguration.SeedWithTestData;
         }
-        
-        private static void RunServer(StartupConfiguration startupConfiguration)
-        {
-            var webHost = CreateWebHostBuilder(startupConfiguration).Build();
-
-            IHostingEnvironment env = (IHostingEnvironment) webHost.Services.GetService(typeof(IHostingEnvironment));
-            IConfiguration conf = (IConfiguration) webHost.Services.GetService(typeof(IConfiguration));
-
-            Startup.SetExceptionsMode(env, conf);
-
-            webHost.Run();
-        }
-
-        private static IWebHostBuilder CreateWebHostBuilder(StartupConfiguration startupConfiguration) =>
-            WebHost.CreateDefaultBuilder(startupConfiguration.Arguments)
-                .UseKestrel()
-                .UseStartup<Startup>()
-                .ConfigureAppConfiguration((builderContext, config) =>
-                {
-                    string dbSettingFile = Path.GetFullPath(Path.Combine(startupConfiguration.ConfigurationDirectoryRoute, "DataBaseConnection.json"));
-                    string mainSettingsFile = Path.GetFullPath(Path.Combine(startupConfiguration.ConfigurationDirectoryRoute, "SunEngine.json"));
-                    string logSettingsFile = Path.GetFullPath(Path.Combine(startupConfiguration.ConfigurationDirectoryRoute, "LogConfig.json"));
-
-                    config.AddJsonFile(logSettingsFile, false, false);
-                    config.AddJsonFile(dbSettingFile, false, false);
-                    config.AddJsonFile(mainSettingsFile, false, false);
-                    config.AddCommandLine(startupConfiguration.Arguments);
-                });
-
-        private static bool SunEngineDllRunServer(StartupConfiguration startupConfiguration)
-        {
-            var webHost = CreateWebHostBuilder(startupConfiguration).Build();
-
-            IHostingEnvironment env = (IHostingEnvironment) webHost.Services.GetService(typeof(IHostingEnvironment));
-            IConfiguration conf = (IConfiguration) webHost.Services.GetService(typeof(IConfiguration));
-
-            if (bool.TryParse(conf["Dev:SunEngineDllRunServer"], out bool sunEngineDllRunServer))
```

Comment:
It's enough to check env.IsDevelopment(). We do not need additional custom variables - it's harder to support and keep track of.